data/conf/infrasim.conf

```diff
@@ -1,59 +0,0 @@
-#This file is used for InfraSIM OpenIPMI configuration file
-
-[main]
-#Supported vnode names:
```

Comment:
I'll vote for what was implemented in infrasim-vxrail, those fields like:
emu_file: chassis/node3/node3.emu
smbios: chassis/node4/node4.bin

feel well if everything being under control. :-)